api-gateway-service/src/main/resources/application.yml

```diff
@@ -27,9 +29,15 @@ resilience4j.circuitbreaker:
       waitDurationInOpenState: 10000
       failureRateThreshold: 60
 
+management:
+  endpoints:
+    web:
+      exposure:
+        include: "*"
+
 spring:
   resources:
-    static-locations: classpath:/static, classpath:META-INF/swagger-ui
+    static-locations: classpath:/static, classpath:META-INF
```

Comment:
Why is the UI in the META-INF folder?
AFAIK That's not a good use for the this folder. See [specification](https://docs.oracle.com/javase/7/docs/technotes/guides/jar/jar.html#The_META-INF_directory)
Exposing the entire `classpath:META-INF` folder seems like an especially bad idea since unlike META-INF/swagger-ui there are not UI files in there.

Reply:
I will investigate and fix

Reply:
From quickly reading https://www.webjars.org/documentation#servlet3 it looks like having static resources in a subfolder of META-INF is a common practice.

I think something like `static-locations: classpath:/static, classpath:META-INF/swagger-ui` is ok but `static-locations: classpath:/static, classpath:META-INF` is not

Reply:
I am sorry, but I don't understand: "static-locations: classpath:/static, ... is ok", but then "static-locations: classpath:/static, ..." is not ok. so "classpath:/static" is ok and is not ok in the same time.

Reply:
oh, got it, it is quote from config. Sorry.

Reply:
Exposing the entire `classpath:META-INF` folder is not a good idea.